explorer/client/cypress/support/commands.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
can we remove this file if it's all comments?

Reply:
Good question, I'll try and let you know if it's possible.

Reply:
Removed